.gitlab-ci.yml

```diff
@@ -790,6 +789,8 @@ github-release:
       if [[ "${CI_COMMIT_TAG:-}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
         mv dist/assets/agent-bundle_linux_amd64.tar.gz dist/assets/agent-bundle_${CI_COMMIT_TAG#v}_linux_amd64.tar.gz
         mv dist/assets/agent-bundle_linux_amd64.tar.gz.asc dist/assets/agent-bundle_${CI_COMMIT_TAG#v}_linux_amd64.tar.gz.asc
+        mv dist/assets/agent-bundle_linux_arm64.tar.gz dist/assets/agent-bundle_${CI_COMMIT_TAG#v}_linux_arm64.tar.gz
```

Comment:
I think it would be helpful to discuss w/ @aunshc before adding this to the release artifacts. We gain a lot from these changes without this addition imo.

Reply:
My initial reasoning for releasing the bundle separately was for users of the standalone `otelcol` binary that also want to use the smart agent extension/receiver.  If this is something that we don't want to support, then I can revert these changes.

Reply:
Got it, missed that. Been chatting with Paulo about a similar scenario for OTel .NET auto instrumentation too - there is a possible use case for this where developers want to try the smart agent receiver/extension on their arm systems. This is not a high priority use case at the moment to be supported, we can revert these changes for now.

Reply:
Thanks @aunshc.  I'll exclude it from the release artifacts.